torch/onnx/utils.py

```diff
@@ -245,6 +245,10 @@ def _model_to_graph(model, args, f, verbose=False, training=False,
                 if i >= user_input_num:
                     inp.setUniqueName(param_names[i - user_input_num])
 
+    input_and_param_names = [val.uniqueName() for val in graph.inputs()]
```

Comment:
let's move this part after `_optimize_graph`

Since `_optimize_graph` may change the name of the inputs (including initializers/parameters)

Reply:
Done.